test/integration/integration_test.go

```diff
@@ -3,19 +3,17 @@
 package integration_test
 
 import (
-	"bytes"
 	"fmt"
 	"io"
-	"io/ioutil"
 	"math/rand"
-	"net/http"
 	"os"
 	"strconv"
 	"testing"
 	"time"
 
 	pb_legacy "github.com/envoyproxy/go-control-plane/envoy/service/ratelimit/v2"
 	pb "github.com/envoyproxy/go-control-plane/envoy/service/ratelimit/v3"
+	"github.com/envoyproxy/ratelimit/src/redis"
```

Comment:
I would prefer that you not use the utility here to compute the limit. I think the tests would be stronger if you hand computed them and then we can check to see if they match?